src/Components/App/App.js

```diff
@@ -90,13 +90,14 @@ const App = () => {
   const resestResults = (event) => {
     setError('')
     event.preventDefault()
-    setVendors(storedVendors)
+    console.log(vendors, 'vendors after rest')
+    console.log(storedVendors, 'vendors after rest')
```

Comment:
Did you mean to leave these console logs in?

Reply:
No i did not

Reply:
can you clear them and push again?

Reply:
Done